src/components/admin.ts

```diff
@@ -8,7 +8,7 @@ const MOD_USER_ID_FOR_BAN_APPEAL: string = vars.MOD_USER_ID_FOR_BAN_APPEAL;
 const makeBanMessage = (reason: string, days?: number): string =>
   `
 Uh oh, you have been banned from the UW Computer Science Club server ${
-    days ? `and deleted their messages in the past ${days} days ` : ``
+    days && `and your messages in the past ${days} days has been deleted`
```

Comment:
`has` -> `have`

Also, while you're at it, it might be worthwhile to use the `pluralize` helper function in the `utils` folder for `days`.

Reply:
hello! thank you for the feedback <3 I added the changes you suggested